distro/openshift/auth/Dockerfile

```diff
@@ -14,14 +14,10 @@ ENV PROXY_ADDRESS_FORWARDING=true
 
 USER jboss
 
-ADD realm.json /tmp/apicurio-realm.json
-ADD auth-entrypoint.sh /opt/jboss/auth-entrypoint.sh
```

Comment:
How is the realm added on startup for a new install with these removed?

Reply:
Realm import added